src/pudl/package_data/notebooks/how_to_use_pudl_out.ipynb

```diff
@@ -0,0 +1,572 @@
+{
```

Comment:
<p>If there's going to be an example in here that requires an EIA API key, we should give them a place to set it as an "environment variable" like thing up here. Or give two options, one which references an environment variable for folks who have set them, and another for if they want to cut-and-paste their key into the notebook.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/catalyst-cooperative/pudl/pull/844/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>good idea!</p>

Reply:
I think we have this at the top of our Catalyst Notebook Template